config/routes.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  get 'posts/index'
+  get 'posts/show'
+  get 'users/index'
+  get 'users/show'
+
```

Comment:
Please remove all these routes configs as you already defined the necessary routes below.